### PR TITLE
Incinerator R-walls

### PR DESCRIPTION
**Pull request by M3hillus**
Changes the regular walls of the incinerator room to R-walls to add to security of the area.

incinerator

Babby's first PR

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
:+1:

**Comment by Earthdivine** [comment]
Wait were those skeleton remains always there?

**Comment by tigercat2000** [comment]
@Earthdivine given that the file changes add it, no

**Comment by Robertha89** [comment]
This is kinda bad, r-walls are supposed to be around secure locations only, security, bridge, engineering, atmos, science and parts of medbay. There is no reason to add more walls, in fact there is a reason to remove a good deal of walls. Walls into space does not need to be r-walls unless previous mentioned location. Since this is an issue with the map, as Someone has been replacing normal walls willy nilly, but of course this happened so long ago.
:-1: NT does not build secure stations, they build deathtraps.

**Comment by Alex6511** [comment]
:+1:

**Comment by M3hillus** [comment]
@Roberttha89 Under normal circumstances I would agree with you. But the recent rash of atoms grief based out of the room pushed @Fox-McCloud to add the atmos access door. I just took it to the next step because all it would take is a welder and a wrench, both of which an aspiring Griffin would have to accomplish their task

**Comment by Markolie** [comment]
Merge conflict, @M3hillus.

**Comment by M3hillus** [comment]
And then I broke it. >_> Baby steps.

File: paradise.dme [hunk 1]
```diff
@@ -5,6 +5,85 @@
...
+#define FILE_DIR "sound/misc"
+#define FILE_DIR "sound/music"
+#define FILE_DIR "sound/piano"
+#define FILE_DIR "sound/turntable"
+#define FILE_DIR "sound/violin"
+#define FILE_DIR "sound/voice"
+#define FILE_DIR "sound/voice/complionator"
+#define FILE_DIR "sound/voice/Serithi"
+#define FILE_DIR "sound/vox_fem"
+#define FILE_DIR "sound/weapons"
```
Comment (TheDZD) [inline paradise.dme]: FILE_DIR needs to be disabled. Open up the Paradise.dme file, go to Build. Peferences for Paradise, and uncheck the "Automatically set FILE_DIR" box.

**Comment by M3hillus** [comment]
I'll bugger with it in a few hours.

**Comment by M3hillus** [comment]
Yeah, looks like I really dun goofed something. Might as well close it and I'll re-do it when I get home.

**Pull request closed without merging**